src/vmm/src/config/mod.rs

```diff
@@ -132,7 +139,38 @@ impl TryFrom<&str> for VcpuConfig {
         Ok(VcpuConfig { num: num.into() })
     }
 }
+/// IrqConfig
+#[derive(Clone, Debug, PartialEq)]
+pub struct IrqConfig {
+    /// Max value of irq.
+    pub max_irq: u32,
+}
+
+impl Default for IrqConfig {
+    fn default() -> Self {
+        IrqConfig { max_irq: MAX_IRQ.into() }
+    }
+}
 
+impl TryFrom<&str> for IrqConfig {
```

Comment:
We don't need this since we are not going to configure the irq from the command line. We should just call `VmConfig::new(&kvm, config.vcpu_config.num, MAX_IRQ)?` instead, and remove the definition of `IrqConfig`.